Library/ViewModels/CommentsViewModel.swift

```diff
@@ -40,6 +40,9 @@ public protocol CommentsViewModelOutputs {
   /// Emits a boolean that determines if comments are currently loading.
   var isCommentsLoading: Signal<Bool, Never> { get }
 
+  /// Emits a boolean that determines if cell separator is to be hidden.
+  var isCellSeparatorHidden: Signal<Bool, Never> { get }
```

Comment:
`cellSeparatorHidden` is typically our convention for outputs related to a view's `hidden` property.

Although - in our case we're actually not hiding it but just setting its color, so here I would in fact emit the color and test that value.

Reply:
Hmm although, does `self.tableView.separatorStyle = .none` not work here?

Reply:
Actually yes, that is a better output. I'll change it to that.